bitnami/postgresql/values.yaml

```diff
@@ -153,3 +153,9 @@ metrics:
     timeoutSeconds: 5
     failureThreshold: 6
     successThreshold: 1
+
+## For use by charts who depend on postrges
+## Allows the depending chart to inject its own init scripts and postgres.conf by overriding the configmaps
```

Comment:
Instead, I would ask for the config map name to be used. That way you don't need to create the config map with the same name (if you don't give a name while installing you don't really know the name in advance).

```
config:
    customInitConfigMapName:
    customConfigMapName:
```

So if you provide a configMapName it will use that, otherwise, it will create one.

Reply:
Of course, this change requires to update the logic in the other files too

Reply:
That makes sense